data/human/ships.txt

```diff
@@ -3133,8 +3133,8 @@ ship "Star Queen"
 		"heat dissipation" .65
 		"fuel capacity" 700
 		"cargo space" 60
-		"outfit space" 360
-		"weapon capacity" 120
+		"outfit space" 366
+		"weapon capacity" 124
```

Comment:
Remove something to make room instead of increasing the space. The Star Queen isn't meant for combat so ditch a Sidewinder.

Reply:
That makes excellent sense. Now that you mention it, having a luxury cruise ship carrying 135 missiles seems... weird. A carrier only has 150, and that's a Heavy Warship. Central missile tube remove, and missiles dropped correspondingly dropped to 90.